src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -120,6 +136,46 @@ public static LastMet parseLastMet(String lastMet) throws ParseException {
         requireNonNull(lastMet);
         String trimmedLastMet = lastMet.trim();
         return new LastMet(trimmedLastMet);
+      
+    }
+
+     * Parses a {@code String RiskAppetite} into an {@code RiskAppetite}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code RiskAppetite} is invalid.
+     */
```

Comment:
Perhaps you're missing the syntax to start a javadoc?
```suggestion
     /**
     * Parses a {@code String RiskAppetite} into an {@code RiskAppetite}.
     * Leading and trailing whitespaces will be trimmed.
     *
     * @throws ParseException if the given {@code RiskAppetite} is invalid.
     */
```